streams/ru.m3u

```diff
@@ -370,8 +366,8 @@ https://sc.id-tv.kz/o.m3u8
 http://ott-cdn.ucom.am/s62/index.m3u8
 #EXTINF:-1 tvg-id="OhotnikirybolovInternational.ru" status="blocked",Охотник и рыболов International (1080p) [Geo-blocked]
 https://strm.yandex.ru/kal/ohotnik/ohotnik0.m3u8
-#EXTINF:-1 tvg-id="Olala.ru" status="timeout",O-la-la (576p) [Not 24/7]
-http://194.9.27.164:8103/play/O_la_la/index.m3u8
+#EXTINF:-1 tvg-id="Olala.ru",O-la-la [Not 24/7]
+http://3dff4cc4.ucomist.net/iptv/KY6S3K7AMQ8FQM/9068/index.m3u8
```

Comment:
<img width="857" alt="image" src="https://user-images.githubusercontent.com/7253922/166911086-dfb91bb1-b162-4fef-bd95-d7f048dc5a07.png">

Reply:
> <img width="857" alt="image" src="https://user-images.githubusercontent.com/7253922/166911086-dfb91bb1-b162-4fef-bd95-d7f048dc5a07.png">
> 

Going to look for an alternative.